spring-kafka/src/main/java/org/springframework/kafka/listener/AbstractMessageListenerContainer.java

```diff
@@ -269,10 +270,6 @@ public boolean isRunning() {
 		return this.running;
 	}
 
-	protected boolean isPaused() {
```

Comment:
Although it is a protected method, the fact that it is not private means that a downstream implementation can technically access it and override it. Therefore, we can't remove the method. We need to deprecate it.